README.md

```diff
@@ -33,6 +33,14 @@ The primary way this is used is through [Mustang] and [Eyra], as their libc
 implementations. It can also be used as a regular library in
 ["coexist-with-libc" mode].
 
+## Runtime requirements
+
+Resolving users and DNS records requires the execution of `getent` which
+should support the arguments `passwd`, `group`, `initgroups`, `services`,
+`ahosts`, `ahostsv4`, and `ahostsv6`. On a regular glibc system the
+`getent` binary is provided by it and uses the NSS setup as usual.
+Similar, a musl system also provides `getent` (but does not use NSS).
```

Comment:
My sense is it's not necessary to list all the `getent` subcommands here. And this list will grow over time if c-ward adds support for more of the NSS APIs; we can just say it needs `getent` as provided on glibc and musl systems.

Reply:
Good, removed the subcommand list.